capstone-project/src/main/webapp/map/map.component.js

```diff
@@ -71,12 +72,24 @@ angular.module('map').component('mapComponent', {
             return containerDiv;
         }
 
+        // Sends a marker to the backend for saving.
+        function postMarker(marker) {
```

Comment:
All other function are anonymous functions, and this one is a regular function definition.
I think it's better if we just choose one way to declare functions and stick to it.